BunnymodXT/modules/ServerDLL.hpp

```diff
@@ -78,6 +79,8 @@ class ServerDLL : public IHookableDirFilter
 
 	entvars_t *obboPushable = nullptr;
 
+	int m_afHealthLast;
```

Comment:
```suggestion
	int lastRecordedHealth;
```
BXT doesn't use the `m_af` notation.